src/pages/custom/TokenItem.tsx

```diff
@@ -29,12 +30,16 @@ const TokenItem = ({ added, onAdd, onRemove, ...props }: Props) => {
   const { token, contract, decimals, ...rest } = props
   const { t } = useTranslation()
 
-  const link = contract && (
-    <FinderLink value={contract}>
-      {truncate(contract)}
-      {!isNil(decimals) && `(${t("decimals")}: ${decimals ?? "6"})`}
-    </FinderLink>
-  )
+  let link
+  if (contract && AccAddress.validate(token)) {
+    link = (
+      <FinderLink value={contract}>
+        {truncate(contract)}
+        {!isNil(decimals) && `(${t("decimals")}: ${decimals ?? "6"})`}
+      </FinderLink>
+    )
+  }
+  // TODO: Link to native token pages.
```

Comment:
there is not going to be a link to native tokens (since they don't have a contract on chain)

Reply:
@alecande11 They could be linked to an external information page on the asset.  Or maybe it wouldn't add too much value for the user.